### PR TITLE
Set node agent upgrade strategy to 25%

### PR DESCRIPTION
**Pull request by mrajashree**
https://github.com/rancher/rancher/issues/24062

### CODE REVIEW

**Comment by prachidamle** [comment]
LGTM

**Comment by dramich** [comment]
All three updated blocks have different formatting and fields, based off the docs they should all match:
```
updateStrategy:
    type: RollingUpdate
    rollingUpdate:
      maxUnavailable: 25%
```

File: pkg/systemtemplate/template.go [hunk 1]
```diff
@@ -337,6 +339,8 @@ spec:
           type: DirectoryOrCreate
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+	  maxUnavailable: 25%
```
Comment (superseb) [inline pkg/systemtemplate/template.go]: This has a tab before `maxUnavailable`

Comment (mrajashree) [reply pkg/systemtemplate/template.go]: The tab was even before the column where updateStrategy starts and doesn't show up in the IDE :/

**Pull request merged**